app/assets/stylesheets/earthworks.css

```diff
@@ -141,4 +141,14 @@ Uncomment lines 142-144
 
 /* .search-query-form .search-btn .blacklight-icons-search {
   display: none;
-} */
\ No newline at end of file
+} */
+
+#main-container .leaflet-container {
+  height: 620px;
+}
+
+@media (max-width: 575.98px) {
```

Comment:
this is `--bs-breakpoint-sm`

Reply:
might be a little more natural to invert it and use `min-width` of `--bs-breakpoint-md`

Reply:
Thanks I was looking for those variables and couldn't find them anywhere.

Reply:
I am trying @media (min-width: var(--bs-breakpoint-md)) {
  #main-container #leaflet-viewer  {
    display: none;
  }
}

and it isn't working. Not sure why

Reply:
sigh. apparently `var()` doesn't work in media queries, but there are proposals to add it to CSS. nevermind